app/controllers/api/v1/student_books_controller.rb

```diff
@@ -8,9 +8,17 @@ def create
     end
   end
 
+  def update
+    student_book = StudentBook.find(params[:id])
+    student_book.update!(student_book_params)
+    render json: StudentBookSerializer.new(student_book)
+  rescue StandardError
```

Comment:
curious why you use the rescue here? does something break?

Reply:
I think it would be unlikely for something to break, but figured it was important to have just in case (e.g. the StudentBook doesn't exist...?) unsure if this is necessary